docs/excel/custom-functions-overview.md

```diff
@@ -268,16 +268,18 @@ To enable the ability to cancel a function, you must implement a cancellation ha
 
 ## Saving and sharing state
 
-Custom functions can save data in global JavaScript variables. In subsequent calls, your custom function may use the values saved in these variables. Saved state is useful when users add the same custom function to more than one cell, because all the instances of the function can share the state. For example, you may save the data returned from a call to a web resource to avoid making additional calls to the same web resource.
+Custom functions can save data in global JavaScript variables, which can be used in subsequent calls. Saved state is useful when users add the same custom function to more than one cell, because all the instances of the function can share the state. For example, you may save the data returned from a call to a web resource to avoid making additional calls to the same web resource.
 
 The following code sample shows an implementation of a temperature-streaming function that saves state globally. Note the following about this code:
 
-- `refreshTemperature` is a streaming function that reads the temperature of a particular thermometer every second. New temperatures are saved in the `savedTemperatures` variable, but does not directly update the cell value. It should not be directly called from a worksheet cell, *so it is not registered in the JSON file*.
+- `streamTemperature` updates the temperature values displayed in the cell every second and it uses the `savedTemperatures` variable as its data source.
 
-- `streamTemperature` updates the temperature values displayed in the cell every second and it uses `savedTemperatures` variable as its data source. It must be registered in the JSON file, and named with all upper-case letters, `STREAMTEMPERATURE`.
+- You'll note `streamTemperature` is a streaming function because it uses a cancellation handler. Although it is not written in this sample for brevity, assume that the JSON metadata for `streamTemperature` specifies `"stream": true` and `"cancelable": true`, both of which are required for streaming functions.
```

Comment:
- You already introduced this list with "Note the following about this code:"  so you don't need to start this bullet with "You'll note...".  

- The first sentence implies that any function that contains a cancellation handler is a streaming function -- this is not true (i.e., it's feasible for a non-streaming function to have a cancellation handler as well).  What's true is that a streaming function must contain a cancellation handler.  Suggest rephrasing the first sentence to: "Because the `streamTemperature` function is a streaming function, it implements a cancellation handler that will run when the function is [canceled](#canceling-a-function)."

- In the code sample, the `streamTemperature` function doesn't implement a cancellation handler -- but it should. You need to add this to the code sample.

- Suggest that you remove the sentence "Although it is not written....".  The point of this section is to explain how state can be shared across multiple calls to a custom function...no need to recap these details about JSON metadata requirements that have already been covered previously in the article. The bulleted list here should be limited to describing things about the following code sample (as "Note the following about this code:" implies).